homeassistant/components/homeassistant/system_health.py

```diff
@@ -0,0 +1,19 @@
+"""Provide info to system health."""
+from homeassistant.components import system_health
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import system_info
+
+
+@callback
+def async_register(
+    hass: HomeAssistant, register: system_health.SystemHealthRegistration
+) -> None:
+    """Register system health callbacks."""
+    register.async_register_info(system_health_info)
+
+
+async def system_health_info(hass):
+    """Get info for the info page."""
+    info = await system_info.async_get_system_info(hass)
+    info.pop("hassio")
```

Comment:
```suggestion
    info.pop("hassio", None)
```

raises KeyError if the key is not present.